scripts/st2bootstrap-deb.sh

```diff
@@ -96,29 +96,54 @@ setup_args() {
   fi
 }
 
+function port_status() {
+  # If a the specified tcp4 port is bound, then return (port,pid,process name) tuple.
+  # Return "Unbound" if a pipe command fails.
+  # Else, return empty string.
+
+  # Use netstat and grep to get a list of all the tcp4 sockets that are in the LISTEN state.
+  # Use cut to ensure that we can grep for the specified port at beginning of the line.
+
+  ret=$(sudo netstat -tunlp4 | grep LISTEN | tr -s ' ' | cut -f 4,7 -d ' ' | cut -f 2 -d ':' | grep ^$1 | tr ' ' '\t' || echo 'Unbound')
```

Comment:
I think the "4" in `-tunlp4` must be a typo. Throwing an error on my machine, and I can't seem to find this option in the man page.